test/test_lm.py

```diff
@@ -0,0 +1,71 @@
+import chainer
+import torch
+
+import lm_chainer
+import lm_pytorch
+
+
+def transfer_lstm(ch_lstm, th_lstm):
+    ch_lstm.upward.W.data[:] = 1
+    th_lstm.weight_ih.data[:] = torch.from_numpy(ch_lstm.upward.W.data)
+    ch_lstm.upward.b.data[:] = 1
+    th_lstm.bias_hh.data[:] = torch.from_numpy(ch_lstm.upward.b.data)
+    # NOTE: only lateral weight can directly transfer
+    # rest of the weights and biases have quite different placements
+    th_lstm.weight_hh.data[:] = torch.from_numpy(ch_lstm.lateral.W.data)
+    th_lstm.bias_ih.data.zero_()
+
+
+def transfer_lm(ch_rnnlm, th_rnnlm):
+    assert isinstance(ch_rnnlm, lm_chainer.RNNLM)
+    assert isinstance(th_rnnlm, lm_pytorch.RNNLM)
+    th_rnnlm.embed.weight.data = torch.from_numpy(ch_rnnlm.embed.W.data)
+    transfer_lstm(ch_rnnlm.l1, th_rnnlm.l1)
+    transfer_lstm(ch_rnnlm.l2, th_rnnlm.l2)
+    th_rnnlm.lo.weight.data = torch.from_numpy(ch_rnnlm.lo.W.data)
+    th_rnnlm.lo.bias.data = torch.from_numpy(ch_rnnlm.lo.b.data)
+
+
+def test_lm():
+    n_vocab = 3
+    n_units = 2
+    batchsize = 5
+    rnnlm_ch = lm_chainer.ClassifierWithState(lm_chainer.RNNLM(n_vocab, n_units))
+    rnnlm_th = lm_pytorch.ClassifierWithState(lm_pytorch.RNNLM(n_vocab, n_units))
+    transfer_lm(rnnlm_ch.predictor, rnnlm_th.predictor)
+    import numpy
+    # test transfer function
+    # numpy.testing.assert_equal(rnnlm_ch.predictor.embed.W.data, rnnlm_th.predictor.embed.weight.data.numpy())
```

Comment:
Why these are commented out?

Reply:
We gave up on transferring weights from chainer to pytorch until onnx. It is OK to add `# TODO(karita) implement weight  transfer`.